Description: drop features depending on packages not in Debian yet
 The crate 'spiffe' has not been packaged yet. As it is an optional dependency,
 disable it for now.

Author: Emanuele Rocca <ema@debian.org>

Last-Update: 2023-06-28
Index: parsec-client/Cargo.toml
===================================================================
--- parsec-client.orig/Cargo.toml
+++ parsec-client/Cargo.toml
@@ -37,10 +37,6 @@ version = "0.4.0"
 [dependencies.parsec-interface]
 version = "0.29.0"
 
-[dependencies.spiffe]
-version = "0.2.0"
-optional = true
-
 [dependencies.url]
 version = "2.2.0"
 
@@ -52,5 +48,4 @@ version = "0.0.3"
 
 [features]
 default = []
-spiffe-auth = ["spiffe"]
 testing = ["parsec-interface/testing"]
